server/grpc_service.go

```diff
@@ -1332,6 +1331,8 @@ func (s *GrpcServer) RegionHeartbeat(stream pdpb.PD_RegionHeartbeatServer) error
 				if cancel != nil {
 					cancel()
 				}
+
```

Comment:
if only switch primary, do we need to close it?

Reply:
Both are ok to me.